dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2292,6 +2293,18 @@
           "type": "number",
           "format": "double",
           "description": "A confidence score between 0 and 1. Scores close to 1 indicate 100% certainty that the identified language is true."
+        },
+        "script": {
+          "type": "string",
+          "description": "Identifies the script of the input document.",
+          "enum": [
+            "Unknown",
+            "Latin"
+          ],
+          "x-ms-enum": {
+            "name": "ScriptValue",
```

Comment:
Recommend "ScriptKind". "Value" doesn't make much sense here, and we want to avoid "Type" as a discriminator.